charts/app-config/templates/env-configmap.yaml

```diff
@@ -30,7 +30,7 @@ data:
   {{- end }}
   GOVUK_SIDEKIQ_JSON_LOGGING: "1"
   GOVUK_WEBSITE_ROOT: https://www.{{ .Values.externalDomainSuffix }}
-  PLEK_UNPREFIXABLE_HOSTS: account-api,asset-manager,feedback,licensify,local-links-manager,locations-api,places-manager,search-api,signon
+  PLEK_UNPREFIXABLE_HOSTS: account-api,asset-manager,feedback,licensify,local-links-manager,locations-api,places-manager,publishing-api,search-api,signon
```

Comment:
Do we need `publishing-api-read-replica` in here too now?

Reply:
Yes, I think we do.  I'll add that now.

Reply:
Yeah, I was going to say the same thing... Feels like a bit of a yak shave to fix that though.